_episodes/03-packaging-installing.md

```diff
@@ -116,129 +117,133 @@ This shows the basic commands and the general options.
 
 ### Step 1: Creating a directory
-Create a directory called **Vehicles**
+Create a directory called **conversions**
 
 ~~~
-mkdir Vehicles
+mkdir conversions
 ~~~
 {: .language-bash}
 
 ### Step 2: Adding Modules
 
+conversions/temp.py
 ~~~
-class Land:
-    def __init__(self):
-        ''' Constructor for this class. '''
-        # Create some vehicles
-        self.members = ['Cars', 'Trucks']
-
-    def printMembers(self):
-        print('Printing members of the Land  class')
-        for member in self.members:
-            print('\t%s ' % member)
+def fahr_to_celsius(temp):
+    return ((temp - 32) * (5/9))
 ~~~
 {: .language-python}
 
-The class has a property named members – which is a list of some vehicles we might be interested in. It also has a method named printMembers which simply prints the list of vehicles of this class.Note: all classes defined must be capable of being imported, and won't be executed directly. this will be saved in a file called **Land.py**
+the file temp.py will be treated as a module called temp. This module contains the function `fahr_to_celsius`. The top level container is the package `conversions`. The end user will import this as:
+`from conversions.temp import fahr_to_celsius` 
 
-Next we create another module named **Water**. Create a file named **Water.py** inside the **Vehicles** directory and add the code below:
 
-~~~
-def isWater(v):
-    waterv =['ship', 'boat']
-    for obj in v:
-        if obj in waterv:
-            print("It is a water vehicle")
-        else:
-            print("It is not a water vehicle")
-~~~
-{: .language-python}
+> ## Exercise
+> 1. Create a file named **speed.py** inside the **conversions** directory and add a function named `kph_to_ms` that will convert kilometres per hour to meters per second:
+>
+> > ## Solution
+> > conversions/speed.py
+> > ~~~
+> > def kph_to_ms(speed):
+> >     return speed / 3.6
+> > ~~~
+> > {: .language-python}
+> {: .solution}
+{: .challenge}
 
-Note the code here is a function that checks which items in a list are water vehicles.
 
 ### Step 3 Adding the init file
 
-Finally, we create a file named `__init__.py` inside the `Vehicles/` directory and add the following code:
+Finally, we create a file named `__init__.py` inside the `conversions` directory and add the following code:
 
 ~~~
-from Land import Land
-from Water import isWater
+import temp
+import speed
 ~~~
 {: .language-python}
 
-The init file is the map that tells python what you package looks like.  It is
-also what makes python know a folder is a module. An empty init file makes a
-folder a module, but by adding code, we can make our package easier to use.
+The init file is the map that tells python what our package looks like.  It is
+also what tells Python a directory is a module. An empty init file marks a
+directory as a module. By adding import code, we can make our package easier to use.
```

Comment:
Does it make sense to demonstrate what init does by trying to use it without? and then add it and use with?

Reply:
This would be take some explaining. I'm pretty sure Python 3.3 + can import from folders without an init. The init file only really becomes essential once we make a package and ship to PyPI. In this case it may be simpler to just always include the init.

Reply:
I was thinking an example of how imports can change with the init, like, reducing the levels of the imports eg how you can use the package.module once you have init or even package.function() if the depending on the contents of the init file? In pilots of this workshop (and in working with students in research ) they tend to struggle with what to put in the init file, other than pattern matching what's already there.